wiki/Beatmap_Editor/en.md

```diff
@@ -1,13 +1,35 @@
 # Beatmap Editor
 
-The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
+The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). This is where all [beatmapping](/wiki/Beatmapping) takes place, and all content for the game is created. Creating content for osu! is, and always will be, free. So feel free to jump right in and start beatmapping!
 
-- [Compose](Compose): The place where a majority of beatmapping happens. This is where you place circles, sliders, and spinners.
-- [Design](Design): The editor for where Storyboards are made; however, most mappers only use the "Background Image/Video" button here.
-- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hitsounds, and hitsound volume.
-- [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This sets the values regarding the title, artist, version name, difficulty, combo colours, and more.
+The beatmap editor is separated into four main sections which can be seen on the top-left corner of the editor. The categories are:
 
-Other components of the beatmap editor include:
+- Compose
```

Comment:
this format of list + sections makes sense for longer documents of this type, but here I think it might be better to just have the paragraph for each part and get rid of the list/sections, since it's so short. or maybe those paragraphs could be the list entries themselves. it doesn't seem useful to have this mini-index when everything fits on a typical screen

Reply:
im unsure how you want me to format this, but I'll try deleting the list and keep them as paragraphs instead